third_party/docfx-doclet-143274/src/main/java/com/microsoft/lookup/ClassLookup.java

```diff
@@ -221,6 +221,12 @@ List<String> determineInheritedMembers(List<ExtendedMetadataFileItem> inheritedM
     return new ArrayList<>();
   }
 
+  @Override
+  public String getStatusComment(TypeElement element) {
+    // Don't provide status comments for classes.
+    return null;

```

Comment:
why would we exclude status comments for classes? we're only including status comments for methods?

Reply:
Correct - because the class templates already provide a beta banner based on the class being marked as beta in the metadata, but don't do the same thing with methods.